content/conference/2019_fosdem_2019.md

```diff
@@ -39,5 +39,37 @@ Belgium<br/>
 
 ## Event report:
 
-TODO :-)
+Once again the Haiku team assembled for FOSDEM. It is the largest open source
+conference, so we have to be there! And it's great meeting the other people too.
 
+This year the booth was run by mmu\_man, oco and PulkoMandy. We had visits from
```

Comment:
Probably this should be a blogpost; and it should likely include the images.

Reply:
Since the pictures are not from me, and the other people attending can complete this, I'm not sure it is really a personal blog post. It's also nice to have some organization on the website and not everything as blogposts, which is a pain to search through when you look for something specific.